thijsvanloef/palworld-server-docker
# Add support for installing the latest Beta version of the Palworld dedicated server

## Context 



Palworld started releasing beta builds, this PR will allow you to install those beta builds using a simple env var.

## Choices



## Test instructions

1. Build image
2. set INSTALL_BETA_INSIDER: true in docker-compose.yml
3. start the server
4. check if the manifest ID changes to the beta build

## Checklist before requesting a review
